code/src/pages/dashboard/dashboard-orders/DashboardOrdersPage.test.tsx

```diff
@@ -113,4 +115,21 @@ describe("OrdersTab", () => {
 
     expect(mockSetIsFilterDrawerOpen).toHaveBeenCalledWith(false);
   });
+
+  test("Should not render pagination if there are one page", () => {
+    renderAndMock();
+
+    const prevPageButton = screen.queryByLabelText("Go to previous page");
+
+    expect(prevPageButton).not.toBeInTheDocument();
+  });
+
+  test("Should not render pagination if there are one page", () => {
```

Comment:
Title of this test is the same as in previous test, despite test outcome is different

Reply:
Fixed)